webapp/updates.py

```diff
@@ -261,8 +271,10 @@ def _build_nevra(self, update_pkg_id):
         arch = self.db_cache.id2arch[arch_id]
         return join_packagename(name, epoch, ver, rel, arch)
 
-    def _process_updates(self, packages_to_process, api_version, available_repo_ids, repo_ids_key, response):
+    def _process_updates(self, packages_to_process, api_version, available_repo_ids,
+                         repo_ids_key, response, module_ids):
         # pylint: disable=too-many-branches
+        module_filter = module_ids is not None
```

Comment:
why is the variable here when it's used only on L335?

Reply:
if it would be defined on L335 that would mean that the evaluation of the expression is done multiple times (inside two for cycles) this way, this expression is evaluated just once

Reply:
kk